english/light-fire/run-away/follow/follow.md

```diff
@@ -1,4 +1,6 @@
 You try to run after the child, noticing that it is covered in melted marshmallows.
 You realize that you also are covered in melted marshmallow goo that is quickly cooling.
 Each step you take gets more and more difficult until all of the goo cools and hardens.
-You are now a tasty statue, nibbled at by mice and birds.
\ No newline at end of file
+You are now a tasty statue, nibbled at by mice and birds.
+
+[To be continued...](english/little-boy/statue/statue.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.